packages/preface/preface.1.1.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+
+maintainer: "xaviervdw@gmail.com"
+authors: [
+  "Didier Plaindoux <d.plaindoux@free.fr>"
+  "Pierre Ruyter <grimfw@gmail.com>"
+  "Xavier Van de Woestyne <xaviervdw@gmail.com>"
+]
+
+license: "MIT"
+tags: ["library" "standard" "monad"]
+homepage: "https://github.com/xvw/preface"
+dev-repo: "git+https://github.com/xvw/preface.git"
+bug-reports: "https://github.com/xvw/preface/issues"
+
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name ] {with-test}
+  [ "dune" "build" "@doc" "-p" name ] {with-doc}
+]
+
+depends: [
+  "ocaml" { >= "4.12.0" }
+  "dune" { >= "2.8.0" }
+  "alcotest" {with-test}
+  "qcheck-core" {with-test & >= "0.18" & < "0.24"}
```

Comment:
`qcheck-core` shouldn't just be a `with-test` dependency as there's a library depending on it AFAICS:
https://github.com/xvw/preface/blob/v1.1.0/lib/preface_qcheck/dune

Also, is the upper 0.24 bound necessary for 1.1.0? It is for 1.0.0, where it was just added in #27473 
When I tested the `master` branch of preface a couple of days ago with the qcheck.0.24 release in #27468
it ran `dune runtest` cleanly... (unless I made a mistake while checking) :shrug:  :thinking:

Reply:
Should be fixed on https://github.com/ocaml/opam-repository/pull/27506/commits/349d5a70120286a1a22b649c741010445c96f2f6 thanks a lot for the explanation (and your previous PR that was a nice remainder to make a new release of Preface :D)